File: responses/WPJ3CB-XN8/reverse_digits.rb
# you can write to stdout for debugging purposes, e.g.
# puts "this is a debug message"

def solution(x)
  sum = 0
  (1..x).each do |num|
    if ((num/3).zero? || (num/5).zero?)
      sum += num
    end
  end
  return sum
end

File: responses/7RXR64-7DZ/reverse_digits.py
# you can write to stdout for debugging purposes, e.g.
# print("this is a debug message")

def solution(x):
    d=[]
    e=0
    for i in range(1,x):
        if i%3==0 or i%5==0:
            d.append(i)
    return sum(d)